StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetVerticalViewController.swift

```diff
@@ -363,6 +363,9 @@ class PaymentSheetVerticalViewController: UIViewController, FlowControllerViewCo
             allowsRemovalOfLastSavedPaymentMethod: configuration.allowsRemovalOfLastSavedPaymentMethod,
             allowsPaymentMethodRemoval: loadResult.elementsSession.allowsRemovalOfPaymentMethodsForPaymentSheet()
         )
+        // TODO(porter) Find a better way
+        var updatedAppearance = configuration.appearance
+        updatedAppearance.paymentOptionView.style = .floating
```

Comment:
This is how we work around the default value in appearance being flatRadio, this now means we need to update tests which test lower level components to update this default. :/

Reply:
I like 5, but upon trying to implement it I found I was just redeclaring much of Appearance and also needed helper instance font scaling functions. For simplicity, I decided to pass an `isEmbedded` flag into `RowButton` for now. It's easy to understand and implement. Happy to talk more about this. I feel like this may be a symptom of Appearance growing too big, or maybe Appearance shouldn't have a `PaymentOptionView` and instead `PaymentOptionView` should be it's own thing and contain an `Appearance` object.

Reply:
> Setting appearance.paymentOptionView.whatever should not have an effect outside of the PaymentOptionView. 
I think the current implementation still has this problem.

> I feel like this may be a symptom of Appearance growing too large, or maybe Appearance shouldn’t define the style or have a PaymentOptionView
I see this problem as coming from the fact that we are trying to reuse PaymentMethodRow for both the embedded list (where we should respect the paymentOptionView appearance API) and not-embedded-list (including the saved PM screen which is notably still used in the embedded integration).

> I think this may indicate that we should reconsider the shape of the public API. IMO, it’s incorrect to have a populated paymentOptionView on Appearance when it’s not being used in embedded. Otherwise, we would need logic to determine when to ignore/modify the API values, like the options we've presented above.

I'm definitely open to making the PaymentOptionView appearance separate from PaymentSheet.Appearance, but it shouldn't be because of our internal implementation.